src/ui/utils/apolloClient.ts

```diff
@@ -27,9 +27,9 @@ export function createApolloClient(token: string | undefined, recordingId: strin
   const httpLink = createHttpLink(token, recordingId);
   const errorLink = onError(({ graphQLErrors, networkError }) => {
     if (graphQLErrors) {
-      console.log(`[Apollo GraphQL error]: ${graphQLErrors}`);
+      console.log("[Apollo GraphQL error]:", graphQLErrors);
     } else if (networkError) {
-      console.log(`[Apollo Network error]: ${networkError}`);
+      console.log("[Apollo Network error]:", networkError);
```

Comment:
not a big deal, but i'm curious why you made this change.

Reply:
This was a drive by fix. Without this, the log shows up as `[Apollo Network error]: [object Object]` which isn't super helpful